Old description was incorrect and apk3 doesn't need replaces anymore
when renaming packages.

Ref https://gitlab.alpinelinux.org/alpine/apk-tools/-/merge_requests/304

--- a/APKBUILD.5.scd
+++ b/APKBUILD.5.scd
@@ -174,8 +174,8 @@ file.
 	virtual provider.
 
 *replaces*
-	Specifies packages that the package replaces. This is typically
-	used for packages renamed by upstream.
+	Specifies packages that the package should replace files from when
+	both are installed.
 
 *replaces_priority*
 	Specifies the numeric value that is used by apk(8) when multiple
